lib/internal/process/finalization.js

```diff
@@ -99,6 +100,7 @@ function createFinalization() {
    * @param {Function} fn
    */
   function register(obj, fn) {
+    emitExperimentalWarning('process.finalization.register');
```

Comment:
What about using the same warning id for all functions so it doesn't print up to three warnings to the user?

Reply:
We can keep three ids if the functions are not intended to be stabilized all at the same time.

Reply:
They are very likely to be stabilized at same time but I still prefer to emit warnings for each one of them, the user is more likely to just use one of those methods in their application